pkg/signalmeow/groups.go

```diff
@@ -176,6 +176,7 @@ type BannedMember struct {
 type GroupChange struct {
 	groupMasterKey types.SerializedGroupMasterKey
 
+	SourceServiceId                    uuid.UUID
```

Comment:
This looks wrong (name doesn't match type and `Id` should be `ID`)

Reply:
Good point, now should be better.